### ISSUE TITLE
discogs tracklist is zero

### ISSUE THREAD

**Issue by xytzw** [jesseward/discogstagger#33]
I'm bunching up 2 issues I encountered into this post and two query/feature request

1. I ran into a track number matching issue -  releases like 998336 for example are reported back as having zero tracks and hence not matched to the total number of tracks in the folder (24 in this case). It seems any track on discog with the return carriage sign is not recognised - I tried another album which had 2 tracks out of 18 marked the same and discotagger reported back 16 tracks in the album instead of 18.

2. I coulnd get the batch feature to work. No matter how I named the text file inside the folder (id or id.txt or id_file) and the release number inside it (id_tag=998336 or 998336 or discog_id=998336) it always asked for the -r parameter. I have the latest version installed.

3. query/feture request
- is there a way to prosess more than one folder at once?
- would it be possible to have an automatic work-around for spaces in the folder names? I found that simply dropping the path to the folder into the terminal doesn't work - and I need to put \ in front of each space which is not ideal.

thanks a lot

**Comment by jesseward** [comment]
Hi @xytzw , i have created a pair of issues as the result of this ticket.

Issue #34 : I introduced this bug during refactoring and have since submitted a fix. Grab the latest source tree (or build 3.0.1)

Issue #35 : I wasn't aware of the `sub_tracks` capability and will take a look at implementing support for this release format.

> is there a way to prosess more than one folder at once?

At the moment no. 

>  I found that simply dropping the path to the folder into the terminal doesn't work - and I need to put \ in front of each space which is not ideal.

If you're tagging from a Linux shell , you can place the directory name in quotes. For example.
```
(03:45 PM) >>> discogs-tagger -s "Pepe Bradock Deep Burnt" -r 204
2017-07-30 15:45:27,410 discogstagger.main INFO     Attempting to tag files from target destination=Pepe Bradock Deep Burnt
2017-07-30 15:45:27,770 discogstagger.discogsalbum INFO     Fetching discogs release. artist=Pépé Bradock & The Grand Brûlé's Choir, title=Burning, id=204
2017-07-30 15:45:27,773 discogstagger.main INFO     Tagging album 'Pépé Bradock & The Grand Brûlé's Choir - Burning'
2017-07-30 15:45:27,774 discogstagger.main INFO     Creating destination directory 'Pepe_Bradock_and_The_Grand_Brules_Choir-Burning-(KIF_S_A_08)-1999-jW'
2017-07-30 15:45:27,774 discogstagger.main INFO     Downloading and storing images
2017-07-30 15:45:31,001 discogstagger.main INFO     Writing file Pepe_Bradock_and_The_Grand_Brules_Choir-Burning-(KIF_S_A_08)-1999-jW/01-Pepe_Bradock_and_The_Grand_Brules_Choir-Burning_Hot.mp3
2017-07-30 15:45:31,047 discogstagger.main INFO     Writing file Pepe_Bradock_and_The_Grand_Brules_Choir-Burning-(KIF_S_A_08)-1999-jW/02-Pepe_Bradock_and_The_Grand_Brules_Choir-The_Right_Way.mp3
2017-07-30 15:45:31,066 discogstagger.main INFO     Writing file Pepe_Bradock_and_The_Grand_Brules_Choir-Burning-(KIF_S_A_08)-1999-jW/03-Pepe_Bradock_and_The_Grand_Brules_Choir-Deep_Burnt.mp3
2017-07-30 15:45:31,086 discogstagger.main INFO     Tagging complete.
```

**Comment by jesseward** [comment]
Closing this ticket as follow up issues were created (or fixed) under #34 , #35 #36 and #37
